File: bestia-models/src/main/java/net/bestia/model/dao/GuildMemberDAO.java
package net.bestia.model.dao;

import java.util.Optional;

import org.springframework.stereotype.Repository;

import net.bestia.model.domain.GuildMember;

@Repository
public interface GuildMemberDAO extends CrudOptionalRepository<GuildMember, Integer> {
	
	Optional<GuildMember> findByPlayerBestiaId(long playerBestiaId);
}

File: bestia-models/src/main/java/net/bestia/model/dao/GuildMemberDAO.kt
package net.bestia.model.dao

import net.bestia.model.domain.GuildMember
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface GuildMemberDAO : CrudRepository<GuildMember, Int> {
  fun findByPlayerBestiaId(playerBestiaId: Long): GuildMember?
}
